doctrine/doctrine-bundle/2.3/config/packages/doctrine.yaml

```diff
@@ -0,0 +1,19 @@
+doctrine:
+    dbal:
+        override_url: true
+        url: '%env(resolve:DATABASE_URL)%'
+
+        # IMPORTANT: You MUST configure your server version,
+        # either here or in the DATABASE_URL env var (see .env file)
+        #server_version: '13'
+    orm:
+        auto_generate_proxy_classes: true
+        naming_strategy: doctrine.orm.naming_strategy.underscore_number_aware
+        auto_mapping: true
+        mappings:
+            App:
+                is_bundle: false
+                type: annotation
+                dir: '%kernel.project_dir%/src/Entity'
+                prefix: 'App\Entity'
+                alias: App
```

Comment:
If a file is identical to a file in a previous version, we symlink the new version file to the old one to avoid duplication :)

Reply:
Waiting on the outcome of https://github.com/symfony/recipes/pull/895 to see where this should go.. I figured that has a better chance of getting merged in before the doctrine pr does.

Reply:
you're totally right - keep it - I didn't see that until now

Reply:
I think we should keep this.